### PR TITLE
Pushing a new proposed look for the main web page, to make it easier …

### PR DESCRIPTION
**Pull request by craiglagegit**
…to access the different images.  Let me know what you think.  I'm not an html expert, so there are surely things that could be cleaner, but it all seems to work.  I have the hooks in there for the mount tracking images, but they won't work yet since we haven't uploaded those images yet.  Here's a screenshot of what it looks like.
Rubin_TV_2_23Jun21

### CODE REVIEW

**Review by SimonKrughoff — APPROVED** [review]
Reviewed.  I have a couple of comments.  The main one is that I don't think the date range search should open another window.

File: src/rubintv/templates/table.html [hunk 1]
```diff
@@ -1,72 +1,169 @@
...
-  </tbody>
-
-</table>
-
-</div>
-
-</body>
\ No newline at end of file
+<link rel="stylesheet" href="/rubintv/static/stylesheets/main_style.css"></link>
+
```
[the remark this answers is not in the document]
Comment (craiglagegit) [reply src/rubintv/templates/table.html:2]: Agreed.  I didn't really understand what it was doing.

File: src/rubintv/templates/table.html [hunk 2]
```diff
@@ -1,72 +1,169 @@
...
+  <label for="date">Begin Date: Format=2021-06-08:</label><br>
+  <input type="text" style="text-align:center;" id="beg_date" name="beg_date" value="2021-06-08"><br>
+  <label for="date">End Date: Format=2021-06-08:</label><br>
+  <input type="text" style="text-align:center;" id="end_date" name="end_date" value="2021-06-10"><br><br>
+  <input type="submit" value="Submit">
+
+  <script>
+    function search_process()
+    {
+    window.open(
```
[the remark this answers is not in the document]
Comment (craiglagegit) [reply src/rubintv/templates/table.html:156]: Agreed.  I went back and forth on this, so I'm happy to keep it in the same window.

File: src/rubintv/templates/table.html [hunk 3]
```diff
@@ -1,72 +1,169 @@
...
+  <input type="text" style="text-align:center;" id="beg_date" name="beg_date" value="2021-06-08"><br>
+  <label for="date">End Date: Format=2021-06-08:</label><br>
+  <input type="text" style="text-align:center;" id="end_date" name="end_date" value="2021-06-10"><br><br>
+  <input type="submit" value="Submit">
+
+  <script>
+    function search_process()
+    {
+    window.open( 
+    "/rubintv/?beg_date=" + document.getElementById("beg_date").value + "&end_date=" + document.getElementById("end_date").value +"&num=1000", "_blank");
```
[the remark this answers is not in the document]
Comment (craiglagegit) [reply src/rubintv/templates/table.html:157]: Yes.

**Pull request merged**